lib/jekyll-seo-tag/image_drop.rb

```diff
@@ -36,15 +36,42 @@ def path
       attr_accessor :page
       attr_accessor :context
 
+      def sprockets
+        @sprockets ||= if context.registers[:site].respond_to? :sprockets
+                         context.registers[:site].sprockets
+                       end
+      end
+
+      def digest_path
+        if sprockets.nil?
+          raise "Jekyll Assets is required"
```

Comment:
I haven't contributed to Jekyll for a long time so I'm not sure if this is the correct approach. Should I use `Jekyll.logger.warn` instead?